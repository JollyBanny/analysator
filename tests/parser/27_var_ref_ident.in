var sum: integer = 0;

begin
    writeln(sum);
end.