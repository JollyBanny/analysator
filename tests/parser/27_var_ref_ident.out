program
└──── program block
      ├──── VAR
      │     └──── var declaration
      │           ├──── sum
      │           ├──── integer
      │           └──── 0
      └──── Compound statement
            └──── Call statement
                  └──── writeln
                        └──── sum
